data/mods/More_Survival_Tools/recipes.json

```diff
@@ -1040,9 +1090,68 @@
     {"id":"HAMMER","level":1,"amount":1}
   ],
   "components": [
-    [ [ "rock", 5 ] ],
+    [ [ "rock", 5 ], [ "ceramic_shard", 1 ], [ "sharp_rock", 1 ] ],
```

Comment:
That would be a copy-paste fail on my part it seems,

Also, 3 superglue is actually what the unmodded recipe for the glass macuahuitl demands, so I might tone that down to 1 super glue. Should I retain the idea of it requiring more bone glue, or not? o3o